pkgs/development/python-modules/pytest-dataset/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pyaml,
+  pyjson,
+  pytest,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-dataset";
+  version = "0.3.2";
+  pyproject = true;
+
+  src = fetchPypi {
```

Comment:
If you want you can ask upstream to tag their releases.

Reply:
Currently looking into how hard it would be to get `pyecotrend-ista` to use `pytest-datadir` instead of  `pytest-dataset`.